NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/RemoveMember.java

```diff
@@ -62,19 +61,16 @@ public boolean execute(CommandSender sender, String[] args) {
 		boolean allowed = false;
 		switch (toBeRemoved){ // depending on the type the executor wants to add the player to
 		case MEMBERS:
-			allowed = perm.isAccessible(t, PermissionType.MEMBERS);
+			allowed = gm.hasAccess(group.getName(), uuid, PermissionType.getPermission("MEMBERS"));
```

Comment:
Actually my comment is more uh, NameLayer local. I'd be okay with scoping the overloaded method so that external plugins can't use it. My issue is more that here we have a native NameLayer command, which already used the group name to getGroup, effectively throwing away that internal lookup to do it again just a few lines later. Internal to NameLayer it makes no sense; externally it's fine to constrain it to the String method call.